athlinks-schema.ttl

```diff
@@ -296,4 +254,16 @@ athlinks:recurrence
 athlinks:scoring
   a owl:ObjectProperty ;
   rdfs:comment "The scoring used on an AthleticEventGroup" ;
-  rdfs:domain athlinks:AthleticEventGroup ;
+  rdfs:domain athlinks:AthleticEventGroup .
+
+athlinks:directions a owl:ObjectProperty ;
+  rdfs:comment "Directions to an event or race." ;
+  rdfs:domain schema:Event ;
+  rdfs:label "Directions" ;
+  rdfs:range schema:Text.
+
+athlinks:streetAddress2 a owl:ObjectProperty ;
+  rdfs:comment "Second line of street address" ;
+  rdfs:domain schema:PostalAddress ;
+  rdfs:label "Street Address 2" ;
+  rdfs:range schema:Text.
```

Comment:
Why not just include a line return in schema:streetAddress? Another property is pretty heavy when a \n could suffice.